recipes/yaclib/all/conanfile.py

```diff
@@ -0,0 +1,159 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps
+from conan.tools.scm import Version
+from conan.tools.files import copy, patch, get
+from conan.errors import ConanInvalidConfiguration
+from functools import lru_cache
+import os
+
+required_conan_version = ">=1.47.0"
+
+
+class YACLibConan(ConanFile):
+    name = 'yaclib'
+    description = 'lightweight C++ library for concurrent and parallel task execution'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://github.com/YACLib/YACLib'
+    license = 'MIT'
+    topics = 'c++', 'async', 'parallel', 'concurrency'
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    exports_sources = 'patches/**'
```

Comment:
There are export conandata patches as well as an apply helper you should be using

Reply:
Used export_conandata_patches